### DISCUSSION TITLE
how to call realtime api from golang?

### DISCUSSION THREAD

**Discussion by Oggy4th** [pocketbase/pocketbase#2577] [Q&A]
how to call realtime api from golang using request?

**Comment by ganigeorgiev** [comment]
You can check https://pocketbase.io/docs/api-realtime/ and https://github.com/pocketbase/pocketbase/discussions/308#discussioncomment-3389399.

**Reply by Oggy4th** [reply]
How to reconnect when unsubscribe?

**Reply by ganigeorgiev** [reply]
I don't understand what do you mean. If you don't have subscription why do you want to reconnect?

**Reply by Oggy4th** [reply]
I mean the connection between the server and client cut off after 5 minutes. How does reconnecting mechanism work?

**Reply by ganigeorgiev** [reply]
That's the idle disconnect. To reconnect you have to send back a new "Connect" request (usually should be handled automatically by the SSE client) and on successfully established connection to resubmit the previous subscriptions (you have to store them somewhere locally as they are discarded server-side once the client was disconnected).
